mongofiles/mongofiles.go

```diff
@@ -467,8 +495,33 @@ func (mf *MongoFiles) Run(displayHost bool) (output string, finalErr error) {
 	case Get, GetID:
 		err = mf.handleGet()
 
-	case Put, PutID:
-		err = mf.handlePut()
+		// If mongofiles --put ... is called, i.e. with multiple supporting
+		// arguments, then add gridFiles specified in mf.FileNameList
+		// (see TOOLS-2667). Otherwise, if mongofiles --put_id is called, then
+		// preserve existing behavior.
```

Comment:
```suggestion
	// If mongofiles --put ... is called, i.e. with multiple supporting
	// arguments, then add gridFiles specified in mf.FileNameList
	// (see TOOLS-2667). Otherwise, if mongofiles --put_id is called, then
	// preserve existing behavior.
```
1. [nit] formatting (extra tabs)
2. [nit] reference to a ticket (see my comment above)
3. It feels awkward to say in a code comment "preserve existing behavior" since the comment will exist beyond the life of this PR. "Preserve existing behavior" doesn't actually summarize what the code _does_, instead it says something about the history of the code which doesn't seem necessary.

Reply:
The formatting is a result of `golangci-lint` -- I didn't specify it this way.

I'll change the content of the comment to remove the ticket name and say something more specific than "preserve existing behavior".